ftest-toxic-frontend-reporter/src/test/java/org/arquillian/cube/q/toxic/reporter/ToxicFunctionalTest.java

```diff
@@ -0,0 +1,76 @@
+package org.arquillian.cube.q.toxic.reporter;
+
+import org.arquillian.cube.HostIp;
+import org.arquillian.cube.impl.util.IOUtil;
+import org.arquillian.cube.q.api.NetworkChaos;
+import org.jboss.arquillian.junit.Arquillian;
+import org.jboss.arquillian.test.api.ArquillianResource;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+
+
+import java.io.IOException;
+import java.net.SocketException;
+import java.net.URL;
+
+import static org.arquillian.cube.q.api.NetworkChaos.DistributedLatencyType.logNormalLatencyInMillis;
+import static org.arquillian.cube.q.api.NetworkChaos.RateType.rate;
+import static org.arquillian.cube.q.api.NetworkChaos.TimeoutType.timeoutInMillis;
+import static org.arquillian.cube.q.api.Q.IterationRunCondition.times;
+
+@RunWith(Arquillian.class)
+public class ToxicFunctionalTest {
```

Comment:
Or if you wanted to run a whole arquillian test and then verify the created report files, then you could get some inspiration from this class https://github.com/arquillian/arquillian-graphene/blob/master/extension/screenshooter/src/test/java/org/jboss/arquillian/graphene/screenshooter/ftest/when/AbstractScreenshotTestCase.java and all classes that inherits this one.
And I still think that this can be done in the same module as the implementation of recorder is - you can just create a new package called "ftest"